### PR TITLE
Support garp injection at epg level

### PR DESCRIPTION
**Pull request by jojimt**

### CODE REVIEW

File: vlanBridge.go [hunk 1]
```diff
@@ -115,6 +140,52 @@ func (vl *VlanBridge) PacketRcvd(sw *ofctrl.OFSwitch, pkt *ofctrl.PacketIn) {
...
 }
 
+func (vl *VlanBridge) backGroundGARPs() {
+	for {
+		vl.garpMutex.Lock()
+
+		workDone := false
+		for _, epgInfo := range vl.epgToEPs {
+			if epgInfo.garpCount <= 0 {
+				break
```
Comment (shaleman) [inline vlanBridge.go]: Should this be a `continue` instead of `break`?

Comment (jojimt) [inline vlanBridge.go]: Yes, thanks for catching it.

**Comment by shaleman** [comment]
Changes look good.
Should we add a unit test for new `InjectGARPs()` API?
Mainly to verify `epgToEPs` map is setup correctly..

**Comment by shaleman** [comment]
LGTM

**Pull request merged**